rfcs/text/0012-orchestrator-field-set.md

```diff
@@ -202,23 +202,30 @@ Examples of source data include:
 }
 ```
 
-<!--
-Stage 2: Included a real world example source document. Ideally this example comes from the source(s) identified in stage 1. If not, it should replace them. The goal here is to validate the utility of these field changes in the context of a real world example. Format with the source name as a ### header and the example document in a GitHub code block with json formatting.
--->
-
 <!--
 Stage 3: Add more real world example source documents so we have at least 2 total, but ideally 3. Format as described in stage 2.
 -->
 
 ## Scope of impact
 
-<!--
-Stage 2: Identifies scope of impact of changes. Are breaking changes required? Should deprecation strategies be adopted? Will significant refactoring be involved? Break the impact down into:
- * Ingestion mechanisms (e.g. beats/logstash)
- * Usage mechanisms (e.g. Kibana applications, detections)
- * ECS project (e.g. docs, tooling)
-The goal here is to research and understand the impact of these changes on users in the community and development teams across Elastic. 2-5 sentences each.
--->
+As this RFC involves the creation of an entirely new fieldset, no breaking
+changes are envisaged. Some existing tooling might need updates to factor in the
+new fieldset's availability, however.
+
+### Ingestion mechanisms
+
+- The Filebeat [Kubernetes processor][5] will need updating, as it currently
```

Comment:
@ChrsMark @jsoriano @kaiyan-sheng Any other integrations or components that might populate these orchestrator fields that we might want to capture here?

Reply:
Same change will be needed in Cloud Foundry and Nomad features. Btw, I see that nomad docs are not available in the public guides, I will take a look.

Reply:
Adding the missing docs for Nomad: https://github.com/elastic/beats/pull/24557

Reply:
++ I also expect Metricbeat Kubernetes module to need updates, in the line of the Filebeat processor/autodiscover provider: https://www.elastic.co/guide/en/beats/metricbeat/current/running-on-kubernetes.html

Reply:
Thanks all - I have added the mentioned services to the document.